cwlcon2024-site/code_of_conduct/index.html

```diff
@@ -27,7 +27,7 @@
 <table class="navigation">
     <tr>
         <td class="navigation">
-            <a class="current" title="Conference Home Page" href="..">Home</a>
+            <a class="current" title="Conference Home Page" href=".">Home</a>
```

Comment:
Does `/` work? I think that's how the home link is normally rendered (except when you have paths in the server, then need to figure out how to handle that…)

Reply:
`/` directs back to "https://www.commonwl.org" when editing the attributes via my local brower's devtools (F12) on the rendered site. `.` points to back to https://www.commonwl.org/conferences/2024/